dali/python/backend_impl.cc

```diff
@@ -28,7 +28,7 @@
 #include "dali/util/half.hpp"
 #include "dali/core/device_guard.h"
 #include "dali/core/python_util.h"
-#include "dali/operators/operators.h"
+#include "dali/operators.h"
```

Comment:
Can it stay in `dali/operators/operators.h`, it helps me understanding that this header belongs to `operators` lib

Reply:
Discussed offline. I figured, that since this will be a part of API, it's better to have it in the same place as `c_api.h`, so that user doesn't have to look around for proper files. And `operators.cc` remained where it is, so that points to proper target